analyzer/analyzer.go

```diff
@@ -131,6 +131,17 @@ func (r *AnalysisResult) Merge(new *AnalysisResult) {
 	defer r.m.Unlock()
 
 	if new.OS != nil {
+		// Alpine contains OS version and repository version
+		// These versions can be different
+		// To OS must add OS version and Repository version
+		if r.OS != nil && r.OS.Family == aos.Alpine && new.OS.Family == aos.Alpine {
```

Comment:
@knqyf263 
It's getting a bit complicated
I would define something like
```golang
type analyzerMerger interface {
	analyzer
	Merge(curr *AnalysisResult, new *AnalysisResult)
}
```
and run all mergers after each analysis in `AnalyzeFile()`.